CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Release report: TBD
 
 Wazuh commit: TBD \
 Release report: TBD
-
+- Add Windows location wildcards tests ([#4263](https://github.com/wazuh/wazuh-qa/pull/4263)) \- (Tests + Framework)
```

Comment:
This line should be in 4.6.0

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4263/commits/6454c401577aed8fee779b0bb2e84d41982708a2